.pylintrc

```diff
@@ -163,6 +163,7 @@ disable=print-statement,
         duplicate-code,
         broad-except,
         no-else-raise,
+        logging-fstring-interpolation,
```

Comment:
we'd like this to happen in another PR. We're also in the process of setting up logging, so this all may change.

Reply:
I have found the logging calls are very helpful for preserving a record of the audit and troubleshooting. f-strings, especially with the new `=` debugging sugar, are great for logging.  But lint complains unless this option is turned on.
So rather than removing or changing all the logging calls, I think it best to keep this one in to accommodate them.
See discussion at https://github.com/PyCQA/pylint/issues/2354#issuecomment-414526879 which doesn't even note the additional benefits provided by the f-string debugging syntax.

Reply:
I'm just worried that this (and the other logging calls) will interfere with our own logging deployment that's currently in the works. Could you at least comment out your logging code so we can be sure that won't happen?

Reply:
If you're worried about the logging config chunk in `config.py`, I can comment that out, and just uncomment locally for my own testing.But leaving the normal calls to log things in place, and the config that allows fstrings, wouldn't seem to affect your updates and deployment, and dealing with all of them during rebases would add burdens.
Unless there's something novel about the way you're deploying it, I don't see any conflicts. The minerva.py logging calls happen rarely, and actually the config changes I made help mask the more verbose logging in the athena module itself.